### ISSUE TITLE
Multi-Application button

### ISSUE THREAD

**Issue by grandpajive** [TilesOrganization/support#2]
There used to be an option to show applications in X columns by Y rows. Now it is a list. I have 22 items in that list and was wondering if you could reimplement that feature.

**Comment by rascarlo** [comment]
@grandpajive yes you are right. such option has be removed to speed up the list creation. Once I am happy with a better implementation it will be re-added. Thanks for opening this issue
